setup.cfg

```diff
@@ -71,3 +71,7 @@ license_files = LICENSE.txt
 python_version = 3.9
 ignore_missing_imports = true
 plugins = numpy.typing.mypy_plugin
+
+[codespell]
+ignore-words-list = coo,nd
+skip = docs/source/changelog.rst
```

Comment:
I think this needs an EOF newline

Reply:
Why is that needed? Locally it looks like `codespell` is picking up the `skip = ` here. Happy to add a newline, I'm just not sure that it's needed

Reply:
It doesn't always matter, but many editors flag files without a trailing newline as an error, since it can be a problem for some tools. I don't like seeing the little red icon, so I fixed it here. See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more info.